plugins/modules/node_settings.py

```diff
@@ -381,7 +379,7 @@ def evaluate_node_query(self):
 
         nodes = self._send_request(
             method='GET',
-            path='/api/latest/nodes' + url_query,
+            path=f'/api/latest/nodes{url_query}',
```

Comment:
I am not sure on how the python compatibility works in modules but I feel like it may break older OS supports as the f-type strings where introduced in python 3.6. Will it works on rhel7 instances where the node may only have python2.7 installed?

Reply:
Fixed.